### ISSUE TITLE
Add conda build for ARM architectures

### ISSUE THREAD

**Issue by camposandro** [cds-astro/cds-healpix-python#31]
I noticed that conda is currently unable to resolve the latest [hats v0.4.0](https://pypi.org/project/hats/) release in ARM-based architectures since no `cds-healpix` conda build exists for ARM Mac/Linux. From your previous releases on [PyPI](https://pypi.org/project/cdshealpix/0.7.0/#files) it seems like it's possible. Do you know if specifying those [build platforms](
https://conda-forge.org/docs/maintainer/conda_forge_yml/#build-platform.) in `conda-forge.yml` would be enough to tackle this issue? Relates to https://github.com/cds-astro/cds-healpix-python/issues/11.

**Comment by ManonMarchand** [comment]
Yes we got a ci failure during the release (see https://github.com/cds-astro/cds-healpix-python/actions/runs/11384977564). One of our dependencies (pyo3) took more than 6h to build. 

It might be potentially fixed when https://github.com/PyO3/maturin/issues/2256 is fixed.

**Comment by ManonMarchand** [comment]
Wait. You have an issue on MAC? This is unexpected, that part did not fail.

**Comment by camposandro** [comment]
Yes, I'm testing on a Mac M2. Installing via pip works like a charm, while `conda install conda-forge::cdshealpix` is unable to resolve the package: `PackagesNotFoundError: The following packages are not available from current channels: - conda-forge::cdshealpix`.

**Comment by fxpineau** [comment]
No file corresponding to your needs [here](https://pypi.org/project/cdshealpix/#files)?
Which MacOS and Python versions do you have?

**Comment by fxpineau** [comment]
True: contrary to pipy, we have never built conda packages (neither cdshealpix nor mocpy) for Mac with ARM processors,
check the list [here](https://conda.anaconda.org/conda-forge/osx-arm64/).
We have to check how to include osx-arm64 to our conda-forge recipes...
Is it blocking for you (or not since you succeed with pypi)?

**Comment by fxpineau** [comment]
Meanwhile, a (unsatisfactory) solution is to create a x64 conda environment, see [this stackoverflow question](https://stackoverflow.com/questions/71515117/how-to-set-up-a-conda-osx-64-environment-on-arm-mac).

**Comment by fxpineau** [comment]
As a first step, [this PR](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6572) on conda-forge must be accepted (not in my hands). So, wait and see....
